component/input/input.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="input.css" />
+    <title>Input</title>
+  </head>
+  <body>
+    <h2>Input section</h2>
+    <section class="temp_container">
```

Comment:
What's temp_container.
Use proper and meaningful naming convention for naming css classes.

Reply:
will make the necessary changes